charts/cloud-connector/README.md

```diff
@@ -14,133 +14,120 @@ $ pre-commit run -a
 
 # Cloud Connector
 
-[Cloud Connector](https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/) - This chart deploys the Sysdig Cloud connector on your Kubernetes cluster to enable threat-detection and image scanning.
+## Overview
```

Comment:
Since this file is automatically updated when a change is done, you have to apply these changes inside the template file (`charts/cloud-connector/README.tpl`).

Reply:
sure. Thank you